src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -441,7 +446,7 @@ def evaluate_checkpoint(
         cfg = load_experiment_cfg(ckpt_root_dir=ckpt_root_dir, experiment_name=experiment_name, run_id=run_id)
 
         add_params_to_cfg(cfg, params=["training_hyperparams.resume=True", f"ckpt_name={ckpt_name}"])
-        cls.evaluate_from_recipe(cfg)
+        cls.evaluate_from_config(cfg)
```

Comment:
Add `return` to not lost the evaluation result pls.

Reply:
Added, [0cfa329](https://github.com/Deci-AI/super-gradients/pull/1904/commits/0cfa329dba084337aa8fa1a15a0a9a3c6aa64bb4)